zhaquirks/sonoff/button.py

```diff
@@ -0,0 +1,76 @@
+"""Device handler for eWeLink WB01."""
+import logging
+
+from zigpy.profiles import zha
+from zigpy.quirks import CustomDevice
+from zigpy.zcl.clusters.general import Basic, Identify, OnOff, PowerConfiguration
+
+from zhaquirks.const import (
+    BUTTON,
+    COMMAND,
+    COMMAND_OFF,
+    COMMAND_ON,
+    COMMAND_TOGGLE,
+    DEVICE_TYPE,
+    DOUBLE_PRESS,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    LONG_PRESS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+    SHORT_PRESS,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class SonoffButton(CustomDevice):
+    """Custom device representing sonoff devices."""
+
+    def __init__(self, *args, **kwargs):
+        """Init."""
+        super().__init__(*args, **kwargs)
+
+    signature = {
```

Comment:
can you add the signature comment here like we have in the other quirks please?

Reply:
Added.